packages/react-composites/tests/browser/call/hermetic/VideoGallery.test.ts

```diff
@@ -255,6 +255,7 @@ test.describe('VideoGallery tests', async () => {
     await pageClick(page, dataUiId(IDS.moreButton));
     await page.locator('span:has-text("View")');
     await page.locator('span:has-text("View")').click();
+    await page.locator('div[role=menuitem]').first().focus();
```

Comment:
So, this is to wait for the menu item to be shown until it proceed?

Reply:
This line is to make sure we are focus on the first menu item so that we take the snapshot on the right and not on the left which happens sometimes.
![image](https://github.com/Azure/communication-ui-library/assets/79475487/0ced18fb-a641-4591-b7a6-9187e73d108f)